Debug App/Tests/Unit Tests/Primer/NolPay/NolPayPhoneMetadataServiceTests.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  NolPayPhoneMetadataServiceTests.swift
+//  Debug App Tests
+//
+//  Created by Boris on 27.10.23..
+//  Copyright © 2023 Primer API Ltd. All rights reserved.
+//
+
+#if canImport(PrimerNolPaySDK)
+import XCTest
+@testable import PrimerSDK
+import PrimerNolPaySDK
+
+final class NolPayPhoneMetadataServiceTests: XCTestCase {
+
+    func testGetPhoneMetadata() {
+        let mockService = MockPhoneMetadataService()
+        let expectedResult = Result<(PrimerValidationStatus, String?, String?), PrimerError>.success((.valid, "+123", "1234567890"))
```

Comment:
Super Nit-pick - I would love `Result<(PrimerValidationStatus, String?, String?), PrimerError>` to be typealiased here to clean up usage down the file.

`typealias ValidationResult = Result<(PrimerValidationStatus, String?, String?), PrimerError>` 
...
`let expectedResult: ValidationResult = .success((.valid, "+123", "1234567890"))`